include/deal.II/non_matching/mesh_classifier.h

```diff
@@ -114,9 +114,9 @@ namespace NonMatching
      * Vector. The triangulation attached to DoFHandler is the one that will be
      * classified.
      */
-    template <class VECTOR>
+    template <class VectorType>
     MeshClassifier(const DoFHandler<dim> &level_set_dof_handler,
-                   const VECTOR &         level_set);
+                   const VectorType &     level_set);
 
```

Comment:
While here, can you remove one empty line here (as there are 2 between this declaration and the next one)?

Reply:
Done!